dockerfiles/run/guice/cassandra-ldap/destination/conf/rabbitmq.properties

```diff
@@ -0,0 +1 @@
+uri=
```

Comment:
why do you need to add these meaningless files?

Reply:
I added the uri has it should be (the binding in the docker will be done when RabbitMQ will be really needed).